docs/core/deploying/native-aot/diagnostics.md

```diff
@@ -0,0 +1,54 @@
+---
+title: AOT Diagnostics
```

Comment:
Should this be in the `deploying/native-aot` folder or in the `diagnostics` folder? If it's here then it should at least be linked from somewhere in the diagnostics folder.

Reply:
I added a link from the diagnostic overview section